mobileversion.html

```diff
@@ -0,0 +1,248 @@
+   <title>Mobile Version Website</title>
+</head>
+
+<body>
+
+   <header class="header">
+      <nav class="nav-bar">
+         <a class="log" href="#">BW</a>
+         
+         <div class="hamburger">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+         </div>
+      </nav>
+   </header>
+   <section>
+      <div class="flex-container">
+         <div class="item5">
+         </div>
+         <div class="item6">
+            <div>
+               <h1 class="mariane">I'm Bryan<br>
+                  Glad to see<br>
+                  you!<br>
+               </h1>
+            </div>
+            <div>
+               <p class="text">
+                  I’m a software developer! I can help you<br>
+                  build a product , feature or website Look <br>
+                  through some of my work and<br>
+                  experience! If you like what you see and <br>
+                  have a project you need coded, don’t<br>
+                  hestiate to contact me.
+               </p>
+            </div>
+            <div>
+               <h2 class="cap">LET’S CONNECT</h2>
+               <ul class="lass">
+                  <li><a href="#"><i class="fab fa-twitter"></i></a></li>
+                  <li><a href="#"><i class='fab fa-linkedin-in'></i></a></li>
+                  <li><a href="#"><i class='fab fa-medium-m'></i></a></li>
+                  <li><a href="#"><i class='fab fa-github'></i></a></li>
+                  <li><a href="#"><i class='fab fa-angellist'></i></a></li>
+               </ul>
+            </div>
+         </div>
+      </div>
+   </section>
+   <section>
+      <div id="Portfolio" class="containe">
+         <div class="item1">
+            <img src="https://cdn.dribbble.com/users/472534/screenshots/6591791/blog_drbl.png" alt="snapshoot">
+            <div class="ton">
+               <h2 class="tonic">Tonic</h2>
+            </div>
+            <div class="sec">
+               <ul class="mainp">
+                  <li class="canopy">CANOPY</li>
+                  <li class="mid">&#8226;</li>
+                  <li class="back">Back End Dev</li>
+                  <li class="mid">&#8226;</li>
+                  <li class="year">2015</li>
+                  </ul>
+            </div>
+            <div class="day">
+               <p class="daily">
+                  A daily selection of privately personalized reads; no accounts or sign-ups required.
+               </p>
+            </div>
+            <div class="maincon">
+               <ul>
+                  <li class="container2"><a href="#">html</a></li>
+                  <li class="container3"><a href="#">css</a></li>
+                  <li class="container4"><a href="#">javaScript</a></li>
+               </ul>
+            </div>
+
+            <div class="sub"><button class="sbmt" type="submit">See Project </button></div>
+

```

Comment:
- [ ] [Optional]You should stick to the design as much as possible. please fix the spacing to stick to the design. See the image below for reference
- [ ] [Optional] You should use the same border for buttons. please implement it
### Your design
![your](https://user-images.githubusercontent.com/81727988/205063284-805184e8-b1c8-41fc-b0ec-4ced51821508.png)
### Figma design
![figma](https://user-images.githubusercontent.com/81727988/205063315-4960c3fc-1547-4a4d-967a-e821c75684f6.png)